.github/workflows/devbuilds.yml

```diff
@@ -128,13 +162,21 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: '3.7'
-        architecture: 'x64'      
+        architecture: 'x64'   
+    - name: Install Git submodules
+      run: |
+        git submodule update --init
     - name: Install Qt
       uses: jurplel/install-qt-action@v2
       with:
         version: 5.14.1
         arch: win64_msvc2017_64
         modules: qtcharts
+    - name: Add msbuild to PATH
+    # doesnt work with windows-2016
+    # uses: microsoft/setup-msbuild@v1.0.0
+      run: |
+        echo "::add-path::C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\MSBuild\15.0\Bin\"
```

Comment:
I am not sure that its required.... Probably it can be removed
Motivation for it was - cmake needs msbuild to compile so probably it should be in a search path and I dont know if its there or not